### DISCUSSION TITLE
Renew Take a lot of time

### DISCUSSION THREAD

**Discussion by ao307** [community/community#142756] [GitHub Education]
### Hi everyone,

My request for academic benefits with GitHub Education has been renewed and approved; however, it has been over three days, and I have not yet received the benefits. I am reaching out to inquire about the status of my request, as I am eager to access the resources available to support my academic journey.

I would appreciate it if you could review my request and ensure that the benefits are provided as soon as possible.

Thank you very much for your assistance and support.

**Comment by lrdcxdes** [comment]
same here

**Comment by thomassiard** [comment]
Double-check Your GitHub Education Dashboard: Log in at [GitHub Education](https://education.github.com/) to see if any updates or pending actions are required on your account.

Check Your Email (Including Spam): GitHub sometimes sends activation details via email, which can sometimes end up in spam or promotional folders.

Contact GitHub Education Support: If you still don’t see the benefits, it’s best to reach out to GitHub Support directly. They can verify your status and ensure everything is correctly set up for your account.

**Reply by ao307** [reply]
The issue has been resolved, thankfully, and everything is working fine now without needing to do anything extra. Thank you!

**Comment by ao307** [comment]
The issue resolved itself after four days. Thank you!
